librocksdb_sys/crocksdb/c.cc

```diff
@@ -2305,10 +2314,16 @@ void crocksdb_options_set_paranoid_checks(
 void crocksdb_options_set_env(crocksdb_options_t* opt, crocksdb_env_t* env) {
   opt->rep.env = (env ? env->rep : nullptr);
 }
-
+crocksdb_logger_t* crocksdb_logger_create(void* rep, void (*destructor_)(void*),logv_cb logv) {
```

Comment:
also add a `crocksdb_logger_impl_destory` function for deleting the pointer.

Reply:
Done